aviary/mission/gasp_based/phases/landing_components.py

```diff
@@ -28,7 +28,9 @@ def compute_partials(self, inputs, J):
 
 class GlideConditionComponent(om.ExplicitComponent):
     def setup(self):
-        self.add_input("rho_app", val=0.0, units="slug/ft**3", desc="air density")
+        self.add_input(
+            Dynamic.Mission.DENSITY, val=0.0, units="slug/ft**3", desc="air density"
```

Comment:
I was surprised to see the 'rho_app' rename in this PR.

Reply:
There was actually an issue where the turboprop needs density (and the old engine models did not). Because this component renames most of the atmosphere outputs, downstream components actually can't access any of them. There appears to be no actual need for these aliases, and it caused problems, so out it goes

Reply:
Can't just alias propulsion inputs because engine decks don't use density, so that would throw an error for those cases